exercise1/exercise3/main.cpp

```diff
@@ -0,0 +1,43 @@
+#include <iostream>
+//Diego Guerrero
+//Exercise 3 - Integral Limits
+//Write a program that will output the upperand lower limits of integral types.
+//Hint: Use #include <climits> to have access to macros that will output these limits.
+
+#include <climits>
+
+void Message(const char* a, int b, int c)
+{
+	//tabla items
+	std::cout << a << "\t\t\t " << b << "\t\t\t  " << c << std::endl;
+}
+
+
+int main()
+{
+	std::cout << "Tabla of integral data type lower and uppr limits in c++" << std::endl;
+
+	std::cout << "---------------------------------------------------" << std::endl;
+
+	std::cout << "Data Type  \t   ----  \tLower Limit \t ----  \tUpper Limit" << std::endl;
+
+	std::cout << "---------------------------------------------------" << std::endl;
+
+
+	Message("signed char", SCHAR_MIN, SCHAR_MAX);
+	Message("unigned char", -UCHAR_MAX / 2, UCHAR_MAX / 2);
```

Comment:
These values are not correct. Why are you making these divisions? What would be the min value of an unsigned that type (a data type that cant be negative)